buildspec-release.yml

```diff
@@ -1,92 +1,34 @@
+      # run sagemaker integ tests
+      #- IGNORE_COVERAGE=- tox -e py27,py36 -- test/integ/sagemaker
 
-      # launch remote gpu instance
-      - prefix='ml.'
-      - instance_type=${GPU_INSTANCE_TYPE#"$prefix"}
-      - create-key-pair
-      - launch-ec2-instance --instance-type $instance_type --ami-name dlami-ubuntu
-      - printf "$SETUP_CMDS" > $SETUP_FILE
-
-      - py2_cmd="IGNORE_COVERAGE=- tox -e py36 -- test/integration/local --docker-base-name $base_name --py-version 2 --framework-version $FRAMEWORK_VERSION --processor gpu"
-      - remote-test --github-repo sagemaker-tensorflow-container --setup-file $SETUP_FILE --branch script-mode --test-cmd "$py2_cmd"
-      - py3_cmd="IGNORE_COVERAGE=- tox -e py36 -- test/integration/local --docker-base-name $base_name --framework-version $FRAMEWORK_VERSION --processor gpu"
-      - remote-test --github-repo sagemaker-tensorflow-container --branch script-mode --test-cmd "$py3_cmd" --skip-setup
-
-      - IGNORE_COVERAGE=- tox -e py36 -- test/integration/sagemaker -n 24 --region $AWS_DEFAULT_REGION --account-id 142577830533 --docker-base-name sagemaker-tensorflow-scriptmode --framework-version $FRAMEWORK_VERSION --processor cpu,gpu --py-version 2,3
-
-      - |
-        echo '[{
-          "repository": "sagemaker-tensorflow-scriptmode",
-          "tags": [{
-            "source": "1.15.0-cpu-py2",
-            "dest": ["1.15.0-cpu-py2", "1.15-cpu-py2", "1.15.0-cpu-py2-'${CODEBUILD_BUILD_ID#*:}'"]
-          },{
-            "source": "1.15.0-cpu-py3",
-            "dest": ["1.15.0-cpu-py3", "1.15-cpu-py3", "1.15.0-cpu-py3-'${CODEBUILD_BUILD_ID#*:}'"]
-          },{
-            "source": "1.15.0-gpu-py2",
-            "dest": ["1.15.0-gpu-py2", "1.15-gpu-py2", "1.15.0-gpu-py2-'${CODEBUILD_BUILD_ID#*:}'"]
-          },{
-            "source": "1.15.0-gpu-py3",
-            "dest": ["1.15.0-gpu-py3", "1.15-gpu-py3", "1.15.0-gpu-py3-'${CODEBUILD_BUILD_ID#*:}'"]
-          }],
-          "test": [
-            "IGNORE_COVERAGE=- tox -e py36 -- -m deploy_test test/integration/sagemaker -n 4 --region {region} --account-id {aws-id} --instance-type {cpu-instance-type} --docker-base-name sagemaker-tensorflow-scriptmode --framework-version 1.15.0 --processor cpu --py-version 2,3",
-            "IGNORE_COVERAGE=- tox -e py36 -- -m deploy_test test/integration/sagemaker -n 4 --region {region} --account-id {aws-id} --docker-base-name sagemaker-tensorflow-scriptmode --framework-version 1.15.0 --processor gpu --py-version 2,3"
-          ]
-        }]' > deployments.json
+      # generate the distribution package
+      - python3 setup.py sdist
 
       # publish the release to github
       - git-release --publish
 
-    finally:
-      # shut down remote gpu instance
-      - cleanup-gpu-instances
-      - cleanup-key-pairs
-
 artifacts:
   files:
-    - deployments.json
+    - dist/sagemaker_pytorch_training-*.tar.gz
```

Comment:
s/pytorch/tensorflow

Reply:
Great catch, i had this change locally but forgot to push it :)